src/pages/api/auth/[...nextauth].js

```diff
@@ -41,13 +41,22 @@ const options = {
           };
         }
       }
+      if (token) {
+        return token;
+      }
       return { ...account, ...user };
     },
+    async session({ session, token }) {
+      if (!(session && token)) {
+        return null;
+      }
+      return { ...session, ...token };
+    },
   },
 
   pages: {
     signIn: "/login",
-    error: "/404", // Error code passed in query string as ?error=
+    error: "/404",
```

Comment:
Looks like a good comment to have & probably for auth errors we should redirect to login or generic error page. 404 is very specific page.

Reply:
currently, we do not have a separate login page. So maybe we may redirect back to the `act-now` page, which has the login. @kilemensi